### ISSUE TITLE
Does Not Install Properly on Ubuntu 16.10

### ISSUE THREAD

**Issue by Starz0r** [modoboa/modoboa-installer#70]
Note: I tried the same setup on a Ubuntu 16.04 setup.

It seems when using the installer script that Nginx cannot bind it's self to the ports set at 80 and 443, and as such, Dovecot cannot startup. I've tried editing the nginx config to see if it can be fixed,  as well as the installer options, but nothing seems to fix this.

I preformed the Install Script with Dependencies installed on a DigitalOcean droplet with 512MB of RAM, 20GB SSD, and Ubuntu 16.10 x64.

Is there any solutions?

**Comment by tonioo** [comment]
Have you tried with more RAM ? The minimum requirement is around 1G if you don't use clamav and 2G if you intend to use it.

Otherwise, have you run the installer using the --debug option ?

**Comment by Starz0r** [comment]
That's a lot of RAM usage, I might try it on my Fedora Workstation which has more RAM, assuming the file paths are the same, it should work?

**Comment by tonioo** [comment]
I think it should work but use the --debug option, you'll see every error that might happen.

**Comment by tonioo** [comment]
@Starz0r ping, any news?

**Comment by Starz0r** [comment]
@tonioo Sorry I haven't gotten back to this in a while.

The Modoboa install seems to work fine on my Home Server, but some lack of documentation on setting it up for a different Webserver basically made me drop it. The installer basically shoves Let's Encrypt and Nginx down your throat, even though I prefer using Caddyserver instead, which does the Let's Encrypt certification for me, automatically.

I checked out Mailcow, but they never seemed to reply to my Github Issue that I opened up almost a week ago, and I'd like to get my own mailserver up and running some how, so I may come back to Modoboa just to see if I can get it setup again if these problems can be resolved. Right now, I'm still mailserver-less.
